### PR TITLE
[new release] rtree (0.1.1)

### PR DESCRIPTION
**Pull request by patricoferris**
A pure OCaml R-Tree implementation

- Project page: <a href="https://github.com/geocaml/ocaml-rtree">https://github.com/geocaml/ocaml-rtree</a>

## Bugs, Fixes and Optimisations

 - Remove extra length calculations (geocaml/ocaml-rtree#16, @lindig)
 - Remove some polymorphic comparisons and replace with `Float` functions (geocaml/ocaml-rtree#15, @patricoferris)
 - Fix stack overflows in OMT and Rectangle.merge (geocaml/ocaml-rtree#14, @patricoferris, reported by @lindig)

### CODE REVIEW

**Comment by mseri** [comment]
Thanks

File: packages/rtree/rtree.0.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+synopsis: "A pure OCaml R-Tree implementation"
+description:
+  "This implements a simple, functional R-Tree library in pure OCaml with support for efficient bulk loading of values."
+maintainer: ["patrick@sirref.org"]
+authors: ["Marius A. Eriksen" "Patrick Ferris"]
+license: "BSD-3-Clause"
+tags: ["spatial" "index"]
+homepage: "https://github.com/geocaml/ocaml-rtree"
+bug-reports: "https://github.com/geocaml/ocaml-rtree/issues"
+depends: [
```
[the remark this answers is not in the document]
Comment (haochenx) [reply packages/rtree/rtree.0.1.1/opam:11]: as this PR is already merged and a dependency to `ocaml` is not added yet.
do you think it's a good idea to open a PR to add `ocaml` to both rtree.0.1.0 and rtree.0.1.1 opam files?

[the remark this answers is not in the document]
Comment (mseri) [reply packages/rtree/rtree.0.1.1/opam:11]: Yes, thanks! Sorry for the oversight

[the remark this answers is not in the document]
Comment (haochenx) [reply packages/rtree/rtree.0.1.1/opam:11]: >Yes, thanks! Sorry for the oversight

Thanks for the response! Here we go:
- https://github.com/ocaml/opam-repository/pull/24291

**Pull request merged**